### ISSUE TITLE
CPU Miner - Benchmarking for >2 hours now…

### ISSUE THREAD

**Issue by redbear-ger** [RainbowMiner/RainbowMiner#2632]
I set up two cpu miners and I’m new to rainbow miner.
Is benchmarking for so long normal?

**Comment by whitindllc** [comment]
in the config you can lower the threshold of time... but to be honest a good through benchmark is way better in the long run

**Comment by RainbowMiner** [comment]
Yes, it does take time. Please make sure, that you do leave at least one thread free for RainbowMiner to run. If you assign all cores/threads to CPU-mining, there will be no more power left to run other system jobs or RainbowMiner.
